File: 1131 - Grenais/grenais.py
grenais = inter = gremio = empates = 0
while True:
    gols_inter, gols_gremio = map(int, input().split())
    grenais += 1
    if gols_inter > gols_gremio:
        inter += 1
    elif gols_inter < gols_gremio:
        gremio += 1
    else:
        empates += 1
    print("Novo grenal (1-sim 2-nao)")
    opcao = int(input())
    if opcao != 1:
        break
print(f"{grenais} grenais")
print(f"Inter:{inter}")
print(f"Gremio:{gremio}")
print(f"Empates:{empates}")
if inter > gremio:
    print("Inter venceu mais")
elif gremio > inter:
    print("Gremio venceu mais")
else:
    print("Nao houve vencedor")

File: 1131 - Grenais/grenais.c
#include <stdio.h>

int main(){
    int grenais = 0, inter = 0, gremio = 0, empates = 0, opcao;
    do {
        int gi, gg;
        scanf("%d %d", &gi, &gg);
        grenais++;
        if(gi > gg) inter++;
        else if(gi < gg) gremio++;
        else empates++;
        printf("Novo grenal (1-sim 2-nao)\n");
        scanf("%d", &opcao);
    } while(opcao == 1);
    printf("%d grenais\n", grenais);
    printf("Inter:%d\n", inter);
    printf("Gremio:%d\n", gremio);
    printf("Empates:%d\n", empates);
    if(inter > gremio) printf("Inter venceu mais\n");
    else if(gremio > inter) printf("Gremio venceu mais\n");
    else printf("Nao houve vencedor\n");
    return 0;
}
